apps/hubspot/src/locations/ConfigScreen.tsx

```diff
@@ -153,11 +154,17 @@ const ConfigScreen = () => {
     <Flex justifyContent="center" alignItems="center">
       <Box className={styles.body}>
         <Heading marginBottom="spacingS">Set up Hubspot</Heading>
-        <Paragraph>
+        <Paragraph marginBottom="spacing2Xs">
           Seamlessly sync Contentful entry content to email campaigns in Hubspot. Map entry fields
           to custom email modules in Hubspot to continuously and automatically keep content
           consistent at scale.
         </Paragraph>
+        <Box marginTop="spacingS" marginBottom="spacing2Xl">
+          <Note variant="neutral">
```

Comment:
I think last time Ryun pointed out that perhaps we wanted to use the Warning variant for the note in [braze QA](https://contentful.atlassian.net/wiki/spaces/ECO/pages/5493588023/Braze+App+V2+Mob+QA). I'd check with Erin/Emily if using the neutral is fine.